### PR TITLE
Added user to pets (:

### PR DESCRIPTION
**Pull request by vyper**

### CODE REVIEW

File: apps/web/controllers/pets/edit.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Edit
     expose :pet
 
     def call(params)
+      # TODO How I can apply more security here?
       @pet = PetRepository.find(params[:id])
```
Comment (fnmunhoz) [inline apps/web/controllers/pets/edit.rb]: Is it possible to filter by `current_user` as you did in the other cases?

Comment (vyper) [inline apps/web/controllers/pets/edit.rb]: Nope. I think that the correct way is create a new query... So, we are forced to do not call method in multiple levels.

Comment (fnmunhoz) [inline apps/web/controllers/pets/edit.rb]: I didn't got the idea, Can you give an example of the problem?

Your point is don't do something like:

```ruby
@pet = PetRepository.all_owned_by(current_user).find(params[:id])
```

I was thinking in something like that:


```ruby
@pet = PetRepository.owned_by(current_user, params[:id])
```
Or do you prefer to do like that? Or something else?

```ruby
@pet = PetQuery.owned_by(current_user, params[:id])
```

Comment (vyper) [inline apps/web/controllers/pets/edit.rb]:
```ruby
# this code don't work :P
# the `find` is ignored
@pet = PetRepository.all_owned_by(current_user).find(params[:id])
```

I think in improve using something like `PetRepository.some_query(current, id)` :)

Comment (fnmunhoz) [inline apps/web/controllers/pets/edit.rb]: Looks good.

`PetRepository` could become big, couldn't it? Maybe PetRepository could delegate the complexity to specific query objetcts, but for now, it is nice like that to me :)

Comment (vyper) [inline apps/web/controllers/pets/edit.rb]: Yehp.. I will return to refactor many points. d:

**Pull request merged**